ui/src/stores/flow.js

```diff
@@ -195,7 +199,7 @@ export default {
                 }
             }
 
-            if (state.isCreating && !overrideFlow) {
+            if (state.isCreating && !overrideFlow && !isExistingFlow) {
```

Comment:
If we are not creating a new flow, we have to be working on an existing flow or is there a third way that I am missing?

I guess what I am asking is why this change?